Lib/test/exception_hierarchy.txt

```diff
@@ -2,7 +2,9 @@ BaseException
  +-- SystemExit
  +-- KeyboardInterrupt
  +-- GeneratorExit
+ +-- BaseExceptionGroup
  +-- Exception
+      +-- ExceptionGroup
```

Comment:
Unfortunately we can't express here that ExceptionGroup subclasses BaseExceptionGroup as well.

Reply:
Add a note in parentheses?